README.md

```diff
@@ -198,6 +198,8 @@ const Example = () => {
 |onIndexChange|`((index: number) => void) \| undefined`||Callback fired when the index changes. It receives the current index.|
 |onTabChange|`(data: { prevIndex: number index: number prevTabName: T tabName: T }) => void`||Callback fired when the tab changes. It receives the previous and current index and tabnames.|
 |pagerProps|`Omit<FlatListProps<number>, 'data' \| 'keyExtractor' \| 'renderItem' \| 'horizontal' \| 'pagingEnabled' \| 'onScroll' \| 'showsHorizontalScrollIndicator' \| 'getItemLayout'>`||Props passed to the horiztontal flatlist. If you want for example to disable swiping, you can pass `{ scrollEnabled: false }`|
+|renderHeaderComponent|`((props: TabBarProps<TabName>) => ReactElement<any, string \| ((props: any) => ReactElement<any, string \| ... \| (new (props: any) => Component<any, any, any>)> \| null) \| (new (props: any) => Component<...>)> \| null) \| undefined`|||
```

Comment:
These types look a bit weird, with the `any`.

@PedroBern please take a look if the documentation looks right.